ansible/playbooks/grouper.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Deploy Grouper
+  hosts: grouper
+  sudo: yes
+  roles:
+      - role: notify-chat
+        username: Grouper
+        icon: ":fish:"
+        msg: "Deploying Grouper"
+
+      - role: grouper
+
+      - role: notify-chat
+        username: Grouper
+        icon: ":fish:"
+        msg: ":heavy_check_mark: Done deploying Grouper"
```

Comment:
Wasn't this just removed by https://github.com/iPlantCollaborativeOpenSource/DE/pull/12?